src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/help-role.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    Specifies the resources role that Jenkins scheduler will receive offers
```

Comment:
As discussed can you rephrase this statement in terms what resources are used for launching tasks (and not in terms of what resources it receives)?